### DISCUSSION TITLE
Set your terminal font to "MesloLGM Nerd Font Mono" - is it mandatory?

### DISCUSSION THREAD

**Discussion by refaelsh** [nvim-neorg/neorg#1243] [Q&A] [answered]
The readme says:

> Set your terminal font to "MesloLGM Nerd Font Mono".

Is this mandatory for correct Norg functionality? I like my terminal with `FiraCode Nerd Font Mono`.

**Comment by champignoom** [comment]
No, it's just a working option.

**Reply by max397574** [reply, marked as answer]
perhaps it should be changed to "a nerd font, e.g. ..."

**Comment by dirkesquire** [comment]
I actually find this wording very confusing. I installed Meslo Nerd Font but task lists are not showing up correctly.
How do I 'set my terminal font'?

Is this a vim command? Do I set this in my init.lua?

**Reply by pysan3** [reply]
You change the terminal's settings.

What terminal emulator do you use?

**Reply by dirkesquire** [reply]
Thanks so much.

I installed MesloLG Nerd Font and Ubuntu Mono Nerd Font on the destination server machine, and I am SSH'd into that server, and inside a TMUX session. I assumed that is correct. I thought that I should set the font on the server side inside of NeoVim/Tmux or the Bash terminal, as opposed to my client machine that runs nothing more than iTerm2 or Termius, but maybe I'm wrong on that. I tried playing with the guitfont setting inside of neovim and setting that to Meslo with "set guifont=MesloLG.h10"  but that had no effect.

I use two main terminals:
- Termius from my iPad to SSH into the server, and then I am inside a TMUX session.
- iTerm2 from my computer to SSH into the server, and then I am inside a TMUX session.

My Nvim version is 0.9.5

The task todo icons don't seem to be rendering properly. Any advice at all would be helpful, thanks!

<img width="1463" alt="image" src="https://github.com/nvim-neorg/neorg/assets/605455/4969ac78-b8d1-4920-b32c-b6621bdbe585">

**Reply by pysan3** [reply]
Remote server configuration has nothing to do with font rendering. What matters is you iterm2/termux config (btw change the font in iterm config, you'll see all neovim and tmux appearance change right? terminal is just sending what _text_ to render, and the client terminal emulator decides _how to display_ those characters).

However, tmux does not have the capability of rendering unicode chars without the `-u` flag.

`tmux kill-server` and then relaunch tmux with `tmux -u` (attach the `-u` on every tmux command you type).
Be careful cuz `tmux kill-server` will kill your unsaved works.
